### PR TITLE
Add blog post on our backlog pruning sessions

### LINKED ISSUE #194: Make a blog post about the backlog pruning sessions
This should cover our thinking in general and our experience from the first two briefly, but mostly include instructions for how to join so that we have something to link to in our communications.

### PR DESCRIPTION
**Pull request by Ainali**
Fixes #194

-----
[View rendered _posts/2021-06-14-pruning-our-backlog.md](https://github.com/publiccodenet/blog/blob/ja-backlog-pruning/_posts/2021-06-14-pruning-our-backlog.md)

### CODE REVIEW

File: _posts/2021-06-14-pruning-our-backlog.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+excerpt: We have a new open process for tackling long standing issues
+categories:
+  - news
+---
+
+# Pruning our backlog - one issue at the time
+
+Two months ago in one of our regular community calls for the [Standard for Public Code](https://standard.publiccode.net), our lead codebase steward for quality, [Eric Herman](https://publiccode.net/who-we-are/team/eric-herman.html), took the lead in tackling some of our oldest issues in the backlog.
+Even though it's a quite hard task, doing issue and git archeology, we felt that it was absolutely worth taking the time for it.
+During just an one hour long session we managed to close two issues as outdated/obsolote, mark two as a [good first issue](https://github.com/publiccodenet/standard/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22), identify two topics for future community calls and give a deadline for another two to close if no reponse was given.
```
Comment (ElenaFdR) [inline _posts/2021-06-14-pruning-our-backlog.md:15]:
```suggestion
During just an one hour long session we managed to close two issues as outdated or obsolote, mark two as a [good first issue](https://github.com/publiccodenet/standard/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22), identify two topics for future community calls and give a deadline for another two to close if no reponse was given.
```

File: _posts/2021-06-14-pruning-our-backlog.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+That is why decided to [do it again last month](https://blog.publiccode.net/community%20call/2021/05/20/pruning-the-oldest-issues.html).
```
Comment (ElenaFdR) [inline _posts/2021-06-14-pruning-our-backlog.md:17]:
```suggestion
That is why we decided to [do it again last month](https://blog.publiccode.net/community%20call/2021/05/20/pruning-the-oldest-issues.html).
```

File: _posts/2021-06-14-pruning-our-backlog.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+author: Jan Ainali
+type: blogpost
+excerpt: We have a new open process for tackling long standing issues
+categories:
+  - news
+---
+
+# Pruning our backlog - one issue at the time
+
+Two months ago in one of our regular community calls for the [Standard for Public Code](https://standard.publiccode.net), our lead codebase steward for quality, [Eric Herman](https://publiccode.net/who-we-are/team/eric-herman.html), took the lead in tackling some of our oldest issues in the backlog.
```
Comment (ElenaFdR) [inline _posts/2021-06-14-pruning-our-backlog.md:13]: What did Eric actually do? Our 'lead steward taking the lead' is weirdly repetitive.

File: _posts/2021-06-14-pruning-our-backlog.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+title: "Pruning our backlog - one issue at the time"
+date: 2021-06-14
+author: Jan Ainali
+type: blogpost
+excerpt: We have a new open process for tackling long standing issues
+categories:
+  - news
+---
+
+# Pruning our backlog - one issue at the time
```
Comment (ElenaFdR) [inline _posts/2021-06-14-pruning-our-backlog.md:11]:
```suggestion
# Pruning the Standard for Public Code backlog - one issue at the time
```

File: _posts/2021-06-14-pruning-our-backlog.md (hunk 3 above)
Comment (Ainali) [reply _posts/2021-06-14-pruning-our-backlog.md:13]: I'll rephrase it!

File: _posts/2021-06-14-pruning-our-backlog.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+author: Jan Ainali
+type: blogpost
+excerpt: We have a new open process for tackling long standing issues
+categories:
+  - news
+---
+
+# Pruning the Standard for Public Code backlog - one issue at the time
+
+Two months ago in one of our regular community calls for the [Standard for Public Code](https://standard.publiccode.net), our lead codebase steward for quality, [Eric Herman](https://publiccode.net/who-we-are/team/eric-herman.html), initiated a review of some of our oldest issues in the backlog.
```
Comment (ElenaFdR) [inline _posts/2021-06-14-pruning-our-backlog.md:13]:
```suggestion
Two months ago during one of our regular community calls for the [Standard for Public Code](https://standard.publiccode.net), our lead codebase steward for quality, [Eric Herman](https://publiccode.net/who-we-are/team/eric-herman.html), initiated a review of some of our oldest issues in the backlog.
```

File: _posts/2021-06-14-pruning-our-backlog.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+Even though it's a quite hard task, doing issue and git archeology, we felt that it was absolutely worth taking the time for it.
+During just an one hour long session we managed to close two issues as outdated or obsolote, mark two as a [good first issue](https://github.com/publiccodenet/standard/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22), identify two topics for future community calls and give a deadline for another two to close if no reponse was given.
```
Comment (ElenaFdR) [inline _posts/2021-06-14-pruning-our-backlog.md:15]:
```suggestion
During just an one hour long session we managed to close two issues as outdated or obsolete, mark two as a [good first issue](https://github.com/publiccodenet/standard/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22), identify two topics for future community calls and give a deadline for another two to close if no reponse was given.
```

**Review by ElenaFdR — COMMENTED** [review]
Very nearly ready to go!

File: _posts/2021-06-14-pruning-our-backlog.md [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+Even though it's a quite hard task, doing issue and git archeology, we felt that it was absolutely worth taking the time for it.
+During just an one hour long session we managed to close two issues as outdated or obsolete, mark two as a [good first issue](https://github.com/publiccodenet/standard/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22), identify two topics for future community calls and give a deadline for another two to close if no reponse was given.
+
+That is why we decided to [do it again last month](https://blog.publiccode.net/community%20call/2021/05/20/pruning-the-oldest-issues.html).
+We made similar progress and felt that we were on to something.
+Besides actually doing the work, we also felt that reviewing them openly, with the possibility for anyone interested to join was a good step for us to do even more work in the open.
+
+Based on that we continue our journey to include our community as much as possible in our work and recently [added this process to our documentation](https://github.com/publiccodenet/about/pull/932).
```
Comment (ElenaFdR) [inline _posts/2021-06-14-pruning-our-backlog.md:21]: stronger causality!

```suggestion
Based on this, we'll continue our journey to include our community as much as possible in our work, so we recently [added this process to our documentation](https://github.com/publiccodenet/about/pull/932).
```

**Review by ElenaFdR — APPROVED** [review]
Beautiful! Ready to go.

**Pull request merged**